packages/shared/components/inputs/AssetAmountInput.svelte

```diff
@@ -22,7 +22,7 @@
     function onClickAvailableBalance(): void {
         /* eslint-disable no-extra-semi */
         /* eslint-disable @typescript-eslint/no-extra-semi */
-        ;({ amount, unit } = parseRawAmount(asset?.balances?.baseCoin.available ?? 0, asset.metadata))
+        ;({ amount, unit } = parseRawAmount(asset?.balance?.available ?? 0, asset.metadata))
```

Comment:
if we need to fix it here, do we also need to update all `balances.baseCoin`?

Reply:
No they are all on the account, david probably did a ctrl f and changed here by accident.